compiler/luci/pass/src/ReplaceNonConstFCWithBatchMatMulPass.cpp

```diff
@@ -142,6 +138,8 @@ bool replace_fc_with_matmul(luci::CircleFullyConnected *fc)
   luci::add_origin(matmul, luci::get_origin(fc));
 
   auto all_zero = [](const luci::CircleConst *c) {
+    if (c->dtype() != loco::DataType::FLOAT32)
+      return false;
```

Comment:
Adding this check to `all_zero` seems not appropriate.

How about simply rejecting non-float const bias? For example, in line 123,

```cpp
auto bc = dynamic_cast<luci::CircleConst *>(fc->bias());
if (bc && bc->dtype() != loco::DataType::FLOAT32)
  return false;
```
This may simplify the codes around line 153.

Reply:
Yes, thanks for suggestion :)

Reply:
as we need to handle `bc == nullptr` case, won't be exactly like this...